src/providers/wfs/qgswfsprovider.cpp

```diff
@@ -978,13 +978,19 @@ bool QgsWFSProvider::deleteFeatures( const QgsFeatureIds &id )
 
 bool QgsWFSProvider::changeGeometryValues( const QgsGeometryMap &geometry_map )
 {
-  //find out typename from uri and strip namespace prefix
+  //find out typename from uri
   QString tname = mShared->mURI.typeName();
   if ( tname.isNull() )
   {
     return false;
   }
 
+  QString namespacePrefix = QgsWFSUtils::nameSpacePrefix( tname );
+  if ( !namespacePrefix.isEmpty() )
+  {
+    namespacePrefix += ":";
```

Comment:
Nitpick mode on, `+= ':'` should be a tiny bit more efficient.

Reply:
Actually considerably more efficient. Last I tested (admittedly with qt4) it was about 1/10th the cost.

Reply:
I took it into account to be a good citizen, but here it is really not a performance critical code path :-)